auslib/web/admin/swagger/api.yaml

```diff
@@ -895,6 +895,33 @@ paths:
         '404':
           $ref: '#/responses/resourceNotFound'
 
+  /releases/{release}/read_only/product/required_signoffs:
```

Comment:
Just to be clear: the reason we want this endpoint (instead of just using /required_signoffs/product), is because we don't want the client to have to parse the entire list of product required signoffs itself?

Reply:
The main reason is because this endpoint holds the very specific logic for products that is no associated with a rule, where the "MAX(ROLE_REQUIRED) + all different roles" for product will be considered.